### PR TITLE
AudioVisualizer component (Bars)

### PR DESCRIPTION
**Pull request by hiroshihorie**
https://github.com/user-attachments/assets/d6ad131e-e3c0-4d89-97e3-0984934da0e7

### CODE REVIEW

File: Sources/LiveKitComponents/UI/Visualizer/Visualizer.swift [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    public let audioTrack: AudioTrack
+
+    @StateObject private var audioProcessor: AudioProcessor
+
+    init(audioTrack: AudioTrack,
+         barColor: Color = .white,
+         barCount: Int = 7,
+         barCornerRadius: CGFloat = 100,
+         barSpacingFactor: CGFloat = 0.015,
+         isCentered: Bool = true)
```
Comment (bcherry) [inline Sources/LiveKitComponents/UI/Visualizer/Visualizer.swift:63]: can you add doc strings to this class? What does isCentered do? I expected it to be used in the UI but it looks like it's just used in the AudioProcessor?

**Review by bcherry — COMMENTED** [review]
code looks good - can you share

File: Sources/LiveKitComponents/UI/Visualizer/Visualizer.swift (hunk 1 above)
Comment (hiroshihorie) [reply Sources/LiveKitComponents/UI/Visualizer/Visualizer.swift:63]: Good point, maybe I should move it to the UI.

Comment (hiroshihorie) [reply Sources/LiveKitComponents/UI/Visualizer/Visualizer.swift:153]: Docs have been added.

**Pull request merged**